src/main/eo/org/eolang/threads/sleep.eo

```diff
@@ -20,8 +20,8 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-+architect yegor256@gmail.com
-+home https://github.com/objectionary/eo
++architect 28lev11@gmail.com
++home https://github.com/objectionary/eo-threads
 +package org.eolang.threads
 +rt jvm org.eolang:eo-runtime:0.0.0
```

Comment:
@levBagryansky Same here, maybe it should be `eo-thread`?

Reply:
@Graur name of repository is eo-threads

Reply:
@levBagryansky But still not eo-runtime, right?